nala/blocks/aside/aside.page.js

```diff
@@ -59,6 +59,7 @@ export default class Aside {
         lightGrey1: 'rgb(238, 238, 238)',
         lightGrey2: 'rgb(245, 245, 245)',
         lightGrey3: 'rgb(249, 249, 249)',
+        lightGrey4: 'rgb(248, 248, 248)',
```

Comment:
This is so close to `lightGrey3`, does that color meet contrast requirements? If so, maybe we can avoid declaring a new color.